specs/BadgeNotifications/BadgeNotifications-spec.md

```diff
@@ -0,0 +1,213 @@
+
+## BadgeNotification Properties
+
+| Name       | Description                                                                                                                                                                                                  | Type                        |
+| ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------- |
+| Expiration | Gets or sets the time after which the badge notification should no longer be displayed. This allows for the badge notification to be time-sensitive and automatically removed when it is no longer relevant. | Windows.Foundation.DateTime |
+
+## BadgeNotificationManager class
+
+The `BadgeNotificationManager` class is the central point for managing badge notifications for an
+application. It provides the functionality to update or clear the badge displayed on the app's tile.
+This class allows developers to modify the badge's appearance by setting a numeric value or a glyph
+and also to remove the badge when it is no longer needed.
+
+## BadgeNotificationManager Properties
+
+| Name    | Description                                                                                                        | Type                     |
+| ------- | ------------------------------------------------------------------------------------------------------------------ | ------------------------ |
+| Default | Provides a default instance of the `BadgeNotificationManager` for use in updating or clearing badge notifications. | BadgeNotificationManager |
+
+## BadgeNotificationManager Methods
+
+| Name   | Description                                                                                                                                                                                                      | Parameters                       | Returns                         |
+| ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------- | ------------------------------- |
+| Update | Applies a change to the badge's glyph or number by providing a `BadgeNotification` object. This method is used to update the badge displayed on the app's tile to reflect a new status or count.                 | BadgeNotification `notification` | void                            |
+| Clear  | Removes the badge notifications for the app from the action center. This method is useful for clearing any badge notifications when they are no longer relevant or when the app wants to reset its badge status. |                                  | Windows.Foundation.IAsyncAction |
```

Comment:
Why does Clear need to return IAsyncAction when Update() is void?  Avoid adding async to the API unless we have a specific requirement.

Reply:
Clear badge has been updated to synchronous as per discussion